src/app/courses/step-view-courses/courses-step-view.component.ts

```diff
@@ -177,4 +178,36 @@ export class CoursesStepViewComponent implements OnInit, OnDestroy {
     });
   }
 
+  surveyExists(): boolean {
```

Comment:
Please fix the indentation through line 212.  Each indentation should be 2 spaces